include/rmm/mr/device/limiting_resource_adaptor.hpp

```diff
@@ -189,6 +189,8 @@ class limiting_resource_adaptor final : public device_memory_resource {
  * @return The new limiting resource adaptor
  */
 template <typename Upstream>
+[[deprecated(
+  "make_limiting_adaptor is deprecated. Use the limiting_resource_adaptor constructor instead.")]]
```

Comment:
*nit*: Should we mention the version in which these were deprecated? (as per the docstring note we suggest for deprecations in cudf-python)

Reply:
Done.